src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -299,7 +299,28 @@ class AppsController(
     }
   }
 
-  private def restartApp(appId: PathId)(implicit identity: Identity): Route = ???
+  private def restartApp(appId: PathId)(implicit identity: Identity): Route = {
+    forceParameter { force =>
+      val newVersion = clock.now()
+      onSuccessLegacy(Some(appId))(
+        groupManager.updateApp(
```

Comment:
Why no `updateAppEither`? 😉

Reply:
It will requite quite a lot of changes :) I think we can have a separate task for such refactoring.